dashboard/app/views/admin_reports/_funometer_percentages_by_day.haml

```diff
@@ -1,20 +1,24 @@
+-# Displays a graph and a table of Fun-O-Meter ratings, by day.
+-# Expects an array of three column arrays [date, percentage, count] in the variable
+-# @ratings_by_day.
+
 .by_day
   %h3
     Percentages by Day
   #chart_by_day
   %table
     %tr
-      - @ratings_by_day_headers.each do |header|
+      - ['Date', 'Percentage', 'Count'].each do |header|
         %th
           %span= header
     - @ratings_by_day.each do |day|
       %tr
         %td
-          %span= day.day
+          %span= day[0]
```

Comment:
As before, I don't see doing so being worthwhile (at least at the moment), though I'm open to doing so if you feel strongly.